src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,50 @@
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  grid-gap: $gap-size;
+  max-width: $max-width-calendar;
+  padding: 50px;
+
+  &__day {
+    box-sizing: border-box;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    padding: 10px;
+    width: $width-calendar__day;
+    height: $height-calendar__day;
+    background-color: #eee;
+    border: 1px solid black;
+    font-family: Arial, sans-serif;
+    font-size: 30px;
+    transition: color 0.5s, transform 0.5s;
```

Comment:
I'm not sure if it's important, but you can also "ease" to each property

Reply:
if i`m correct, 'ease' is default

Reply:
Just checked, you are right👍 Thanks for info